### PR TITLE
Added Responsiveness

### LINKED ISSUE #3: Make Website Mobile Responsive
Cards are getting cropped up in mobile view

image

Also Hamburger Menu-Icon gets onto the second row
image

### PR DESCRIPTION
**Pull request by TheCodinggDude**
Fixes #3

### CODE REVIEW

File: views/nav.ejs [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
         <li class="nav-item">
           <a class="nav-link" href="/discover">Trending</a>
         </li>
-       
-        
       </ul>
-      <form class="d-flex" action="/search" method="POST">
-        <input class="form-control me-2" name="searchTerm" type="search" placeholder="Search a Title" aria-label="Search">
-        <button class="btn btn-outline-success" type="submit">Search</button>
+      <form class="form-inline my-2 my-lg-0" action="/seacrh" method="POST">
```
Comment (akk312000) [inline views/nav.ejs:23]: Please fix the typo here.

**Review by akk312000 — CHANGES_REQUESTED** [review]
Hey @TheCodinggDude ...thanks for the PR..everything looks nice now, I just had one more request,
can you also please fix this overflowing issue in the genre section
image

**Comment by TheCodinggDude** [comment]
Sure thing. Will submit the final PR by evening.

**Review by akk312000 — APPROVED** [review]
Thank you for the PR @TheCodinggDude , everything looks good to me now 🚀.

**Pull request merged**